package.json

```diff
@@ -13,7 +13,8 @@
     "axios": "^0.19.0",
     "bottleneck": "^2.19.0",
     "lodash": "^4.17.11",
-    "notifications-node-client": "^4.6.0"
+    "notifications-node-client": "^4.6.0",
+    "zlib": "^1.0.5"
```

Comment:
Isn't `zlib` part of the node standard library?

Reply:
Removed